acvm-repo/acvm/src/pwg/brillig.rs

```diff
@@ -29,6 +29,10 @@ pub enum BrilligSolverStatus<F> {
 pub struct BrilligSolver<'b, F, B: BlackBoxFunctionSolver<F>> {
     vm: VM<'b, F, B>,
     acir_index: usize,
+    /// This id references which Brillig function within the main ACIR program we are solving.
+    /// This is used for appropriately resolving errors as the ACIR program artifacts
+    /// set up their Brillig debug metadata by function id.
+    function_id: u32,
```

Comment:
```suggestion
    function_id: BrilligFunctionId,
```
We should avoid raw integer types for IDs

Reply:
done, I also replaced other usage of u32s for brillig function ids